lib/fly.js

```diff
@@ -315,7 +315,7 @@ Fly.prototype.target = function (dirs, options) {
 				for (var filter of _filters) {
 					// run the filter's closure fn w/data
 					var res = yield Promise.resolve(filter.cb.apply(
-						self, [data, assign({filename: f.base}, filter.options)].concat(filter.rest)
+						self, [data, assign({file: f}, filter.options)].concat(filter.rest)
```

Comment:
I'm thinking that as a 1.3 release, it'd be okay to change something like this. Right?

@bucaran to verify